docs/general/builtins/aggregation.rst

```diff
@@ -825,6 +825,28 @@ Example::
     executions of the aggregate function on the same data produce slightly
     differing results.
 
+.. _aggregation-top_k:
+
+``top_k(column, limit)``
+------------------------
+
+The ``top_k`` aggregate function computes an approximation of the most frequent
+values and their frequencies of a specific column. The result is an array
+containing the most common values and their frequencies. The limit is an optional
```

Comment:
Thoughts on returning an object instead of an array? I think that would make the value/frequency separation clearer and it is easier to work with the reuslt.